src/rnp/fficli.cpp

```diff
@@ -1347,109 +1347,59 @@ cli_rnp_keys_matching_strings(cli_rnp_t *                     rnp,
  *
```

Comment:
looks like `NULL` in description is no longer appropriate

Reply:
As documentation of other parameters :) Thanks, will update the PR.